### PR TITLE
Further improve button text based on 'Instant Booking' status

### PR DESCRIPTION
**Pull request by salvoscala**
Two tasks:

- [x] Casa Properties - Remove the unnecessary 'field_st_allow_instant_bookings' from the House Type bundle
- [x] Locanda Properties - Alter the button text in the 'Property Types details' view (/admin/structure/views/view/property_travel/edit/panel_pane_3) based on the setting in the 'field_st_allow_instant_bookings' field.

### CODE REVIEW

**Review by cecrs — COMMENTED** [review]
looks good!

File: modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.module [hunk 1]
```diff
@@ -919,8 +919,9 @@ function roomify_accommodation_booking_confirmation_form($form, &$form_state, $s
     if (variable_get('roomify_accommodation_booking_allow_instant_bookings', 1)) {
       if (isset($property->field_sp_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
           $property->field_sp_allow_instant_bookings[LANGUAGE_NONE][0]['value']) {
-        if (isset($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
-            $type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) {
+        // If it's a home we don't need to go over.
+        if ($type->type == 'home' || (isset($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
```
Comment (acrollet) [inline modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.module]: can we replace the two tests with a single !empty($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) ? Also I'm not quite sure what the comment means :)

File: modules/roomify/roomify_listing/roomify_listing.fields.inc [hunk 2]
```diff
@@ -2645,43 +2645,46 @@ function roomify_listing_create_standard_type_fields($type_bundle) {
...
         ),
-      ),
-      'description' => 'Enabling instant bookings means that users will be able to book and pay immediately. Make sure to setup a suitable payment method.',
-      'display' => array(
-        'default' => array(
-          'label' => 'above',
-          'module' => 'list',
-          'settings' => array(
-            'conditions' => array(),
+        'description' => 'Enabling instant bookings means that users will be able to book and pay immediately. Make sure to setup a suitable payment method.',
```
Comment (acrollet) [inline modules/roomify/roomify_listing/roomify_listing.fields.inc:2657]: is this text translatable?

**Review by acrollet — CHANGES_REQUESTED** [review]
minor feedback

File: modules/roomify/roomify_listing/roomify_listing.fields.inc (hunk 2 above)
Comment (salvoscala) [reply modules/roomify/roomify_listing/roomify_listing.fields.inc:2657]: @acrollet I think the problem is not just here, I tried to create a new boolean field and have the same issue: it's impossible to translate the description.

Comment (salvoscala) [reply modules/roomify/roomify_listing/roomify_listing.fields.inc:2657]: @acrollet I added a patch to the Token module to solve this known issue. Now label and descriptions of fields are completely translatable :)

**Pull request merged**